sigs.yaml

```diff
@@ -1002,6 +1002,20 @@ sigs:
       url: https://zoom.us/j/861487554?pwd=dTVGVVFCblFJc0VBbkFqQlU0dHpiUT09
       archive_url: https://docs.google.com/document/d/1LdooNTbb9PZMFWy3_F-XAsl7Og5F2lvG3tCgQvoB5e4/edit
       recordings_url: https://www.youtube.com/playlist?list=PL69nYSiGNLP29D0nYgAGWt1ZFqS9Z7lw4
+  - name: cluster-api-operator
+    contact:
+      slack: cluster-api
```

Comment:
Should we get a dedicated channel?
The same applies to meeting URL/time slot and meeting minutes

Reply:
I thought we can share the same channel and meeting for updates/discussions as the operator subproject is tightly related to the core Cluster API. If others have objections, we have an operator channel/meeting.

Reply:
I'm not sure this will work in the long term, but let's hear other opinions.

If we are down this patch I would avoid duplicating meetings info, so there is no need to keep them in sync. Is it possible/enough to put a note that the project is using cluster API channels somewhere?

Reply:
@neolit123 opinions?

Reply:
i don't a strong opinion on this.
for a WIP project a separate channel makes more sense.